packages/server/__tests__/common.ts

```diff
@@ -0,0 +1,148 @@
+require('../../../scripts/webpack/utils/dotenv')
+import axios from 'axios'
```

Comment:
are you opposed to using `node-fetch` over axios? would love to continue using a single package for all HTTP reqs so we don't have to worry about different APIs & keeping both up to date wrt security stuff.